Automattic/jetpack
# Settings: Add toggle for Widget visibility under Widgets card in the Writing tab

Fixes #11936 
Depends on #12004 

#### Changes proposed in this Pull Request:

* Adds toggle for Widget Visibility in the Writing tab reading:
    `Control where widgets appear on your site with visibility settings`.
* Adds info popover text:
    `Configure widgets to appear only on certain posts or pages.`


#### Screenshot

![image](https://user-images.githubusercontent.com/746152/56212896-51969d80-6031-11e9-9577-628f8a60ef65.png)


#### Testing instructions:

* Checkout this branch and run yarn build or [launch a JN site with this branch](https://jurassic.ninja/create?jetpack-beta&branch=add/widget-visibility-toggle&wp-debug-log).
* Visit the settings page and the Writing tab.
* Confirm you see a new card for Widgets.

#### Proposed changelog entry for your changes:

* Admin Page: Added toggle for activating the Widget Visibility module.
